.travis.yml

```diff
@@ -63,4 +64,6 @@ script:
   - make
   - make check
   - make deb
+  - docker build -f docker/Dockerfile -t centos:7 .
+  - docker run --rm -v $PWD:/home centos:7 make docker-rpm
```

Comment:
>  IMO the approach you have here is complicated by the fact you are not using build matrix. I would strongly recommend to go with the build matrix, then update this P.R. based on it.

Wrt. the RPM build to the travis script I am not sure much is gained by using the matrix. Building the Dockerfile and running the container still needs to be done explicitly in the `script` section. 
I can integrate the matrix into this PR and move the RPM build commands to the aux script as I have done in #162, so it is prepared for merging with #162 but again IMHO solely wrt to the RPM build it won't make much of a difference